app/assets/stylesheets/common/_markus.scss

```diff
@@ -1339,3 +1339,12 @@ canvas {
 .required-files-container {
   display: flex;
 }
+
+//Styling for the text on any error pages (for eg. 404, 403)
+.error-pages-container {
+  left: 50%;
```

Comment:
Okay, so overall I think you've made the CSS harder for yourself than you need to. You can remove all of the rules here except the `text-align` (which does the text centering).